docs/source/markdown/podman-create.1.md

```diff
@@ -22,6 +22,35 @@ Default settings for flags are defined in `containers.conf`. Most settings for
 remote connections use the server's containers.conf, except when documented in
 man pages.
 
+## IMAGE
+
+ The image is specified using transport:path format. If no transport is specified, the `docker` (container registry)
+ transport will be used by default. For remote Podman, `docker` is the only allowed transport."
+
+  **dir:**_path_
+  An existing local directory _path_ storing the manifest, layer tarballs and signatures as individual files. This
+  is a non-standardized format, primarily useful for debugging or noninvasive container inspection.
+
+  **docker://**_docker-reference_
+  An image reference stored in  a remote container image registry. Example: "quay.io/podman/stable:latest". 
+  The reference can include a path to a specific registry; if it does not, the registries listed in 
+  registries.conf will be queried to find a matching image. By default, credentials from podman login (stored at 
+  $XDG_RUNTIME_DIR/containers/auth.json by default) will  be used to authenticate; if these cannot be found, we 
+  will fall back to using credentials in $HOME/.docker/config.json.
+
+  **docker-archive:**_path_[**:**_docker-reference_]
+  An image stored in the `docker save` formatted file.  _docker-reference_ is only used when creating such a
```

Comment:
This is still really unclear about `docker-reference` - if it's only used when it created, why do we want it here when we're only pulling/using such an image?